render-test/runner.cpp

```diff
@@ -48,6 +89,12 @@ const std::vector<std::string>& TestRunner::getPlatformExpectationsPaths() {
     return result;
 }
 
+// static
+gfx::HeadlessBackend::SwapBehaviour swapBehavior(MapMode mode) {
+    return mode == MapMode::Continuous ? gfx::HeadlessBackend::SwapBehaviour::Flush
```

Comment:
glFlush significantly affected (halved if I remember correctly) fps count using iOS benchmark app on on iOS iPad 2, and it got removed in
https://github.com/mapbox/mapbox-gl-native/pull/14907.
How much does it affect the measured results here? Is there a way not to use it?

Reply:
There's no implicit flush operation (like swapping buffers) in the headless backend (afaik) which means it's not guaranteed that gpu commands get executed during the right frame. This might lead to wrong results when measuring time spent on rendering. This is not an issue when rendering to the screen or reading framebuffer contents from the gpu.

Manual flushing in the Swap function mimics the behavior of swapBuffers or reading still image from the framebuffer. This might not be necessary here since we're mostly interested in relatice changes in the rendering performance.